workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -55,17 +65,111 @@ private void OnDisconnect(DisconnectOp op)
             view.Disconnect(op.Reason);
         }
 
+        private void OnAddComponent(AddComponentOp op)
+        {
+            if (!componentSpecificDispatchers.TryGetValue(op.Data.ComponentId, out var specificDispatcher))
```

Comment:
👍 C# 7

Reply:
Is that allowed in latest Unity?

Reply:
Incremental compiler, which is required, is C# 7